about/package-scope.md

```diff
@@ -30,9 +28,229 @@ open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSc
 one of our editors. We are happy to look at your package and help you understand
 whether it is in scope or not.
 
-```{include} /appendices/scope.md
+## About the scope of packages that we review
+
+pyOpenSci supports packages that support both open reproducible science,
+data processing and and the various stages of managing the
+data lifecycle. Packages submitted to pyOpenSci should fit into one or
+more of the categories below.
+
+```{admonition} Package Use Metrics Are Not a Requirement for Review
+:class: important
+pyOpenSci has no requirements surrounding the volume community use of your
+package.
+If your package is well developed, the number of GitHub stars or PyPI or conda
+downloads is NOT considered as a part of our scope evaluation.
+
+We welcome young packages that are just entering the scientific Python
+ecosystem to apply for review if they are relevant to the science community and
+fit into at least one scope category below.
 ```
 
+If you are unsure whether your package fits into one of the general or
+statistical categories, please open an issue as a pre-submission inquiry (Examples). <todo add link>
+
+```{note}
+This is a living document. The categories below may change through time.
+This may mean in some cases, some previously peer review-accepted packages
+may not be in-scope today. We strive for consistency in our peer review process. However, we also evaluate packages on a case-by-case basis.
+In some cases exceptions are made.
+```
+
+## Package scope categories
+
+The following categories are the current domain areas that fall into the
+pyOpenSci domain scope. Note that your package should have some level of
+demonstrated scientific application. This could be a use case that you can
+link to or a tutorial that demonstrates its potential application for science.
+
+Below we provide examples of packages from pyOpenSci ecosystem. Because we
+have growing community of packages, in some cases we link to R packages
+within the rOpenSci community that match the category scope for reference.
+
+We will update this page as our review process evolves.
+
+```{note}
+Many of the example packages below perform tasks that might fit in multiple
+categories. There are just there to provide you with a flavor of the types
+of packages that would fall into that category.
+```
+
+### Data retrieval
+Packages for accessing and downloading data from online sources. This category
+includes wrappers for accessing APIs.
+
+Our definition of scientific applications is broad, including data storage
+services, journals, and other remote servers, as many data sources may be of
+interest to scientists. However, retrieval packages should be focused on data
+sources / topics, rather than services.
+
+*For example a general client for Amazon Web Services data storage would not be in-scope.*
+
+* Examples: [OpenOmics](https://github.com/pyOpenSci/software-submission/issues/31)
+
```

Comment:
Could add PyDOV as an example here too, and physcraper (repeating myself, sry)

Reply:
no worries - i'm gonna push a bunch of these edits and then ask you to go through and add these elements!! i'd appreciate the help :) !